src/QueryBuilder.js

```diff
@@ -3,7 +3,11 @@
 var _ = require('lodash')
   , Promise = require('bluebird')
   , RelationExpression = require('./RelationExpression')
-  , ValidationError = require('./ValidationError');
+  , ValidationError = require('./ValidationError')
+  , utils = require('./utils')
+  , jsonFieldExpressionParser = require('./jsonFieldExpressionParser')
+  , knex = require('knex')
```

Comment:
knex is not a dependency anymore. Use `this.modelClass().knex()` to get the knex object. The websql build will fail if you leave this here.